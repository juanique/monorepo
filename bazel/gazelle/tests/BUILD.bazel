load("@bazel_gazelle//:def.bzl", "gazelle_binary", "gazelle_generation_test")

# gazelle:exclude ./

filegroup(
    name = "test_data",
    srcs = glob(["**/*"]),
)

gazelle_binary(
    name = "gazelle_binary",
    languages = [
        "@bazel_gazelle//language/go",
        "//engprod/gazelle/ts",
    ],
)

gazelle_generation_test(
    name = "gazelle_plugins_test",
    gazelle_binary = ":gazelle_binary",
    test_data = [":test_data"],
)
