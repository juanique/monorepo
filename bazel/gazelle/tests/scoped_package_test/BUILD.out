load("//bazel/ts:defs.bzl", "ts_library")

ts_library(
    name = "scoped_package_test",
    srcs = ["test.ts"],
    deps = ["//:node_modules/@playwright/test"],
)
